packages/docusaurus-plugin-sitemap/src/createSitemap.ts

```diff
@@ -19,12 +19,17 @@ export default async function createSitemap(
   if (!hostname) {
     throw new Error('URL in docusaurus.config.js cannot be empty/undefined.');
   }
-  const {changefreq, priority} = options;
+  const {changefreq, priority, ignorePatterns} = options;
 
   const sitemapStream = new SitemapStream({hostname});
 
   routesPaths
-    .filter((route) => !route.endsWith('404.html'))
+    .filter(
```

Comment:
should we apply trailingSlash before the filtering? I don't know what is best in this case 🤷‍♂️

Reply:
404s should not be applied trailing slash anyways, I think this is fine. The odds that someone gets tricked by this seems low to me, and they can always check their glob.